.github/workflows/create_conda_envs.yml

```diff
@@ -12,8 +12,8 @@ env:
   METPLUS_ENV_VERSION: v6.1
```

Comment:
If this is a METv12.0.3 bug fix, should it be METplus v6.1 or v6.0 (I can never keep it straight!)